src/algebra/group.lean

```diff
@@ -5,7 +5,7 @@ Authors: Jeremy Avigad, Leonardo de Moura
 
 Various multiplicative and additive structures.
 -/
-import tactic.interactive data.option.defs
+import tactic.ext tactic.simpa tactic.interactive data.option.defs
```

Comment:
I haven't checked what's going on here, but did you intend to leave `tactic.interactive` in the imports? I thought the point was to remove it.

Reply:
Sorry, I realise I misunderstood what's going on here. How about we even change the name of `tactic.interactive`? I think it might be confusing for people who have previously been used to using `import tactic.interactive` to "get everything".